Winget-AutoUpdate-Install.ps1

```diff
@@ -231,7 +230,7 @@ function Install-WinGet {
 
         #Install/Update Winget
         Update-WinGet $WinGetAvailableVersion
-
+        Write-ToLog "-> WinGet v$WinGetInstalledVersion installed!`n" "Green"

```

Comment:
Function `Update-WinGet` already has `Write-ToLog` for all states (also **success**) in it:
```
    $download_string = "-> Downloading WinGet MSIXBundle for App Installer..."
    $install_string = "-> Installing WinGet MSIXBundle for App Installer..."
    $success_string = "-> WinGet MSIXBundle (v$WinGetAvailableVersion) for App Installer installed successfully"
    $reset_string = "-> WinGet sources reset."
    $fail_string = "-> Failed to install WinGet MSIXBundle for App Installer..."

```

Reply:
Thanks for your review :)

Reply:
...only missing a **CR** if that is of importance?

Reply:
?

Reply:
You had:
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/5b7e82c2-2af2-4b63-a772-9e4d070c73e4)

Reply:
Put back. Is that good now? :)

Reply:
Haha, it's not me that wanted that (I don't se why), I just thought you thought it was important...
...and now you've put it on just two lines (3 missing) 😁:
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/4b1435a2-1650-4a7f-bfa2-c94c4f2b81d7)

Reply:
I'll won't interfera again 😉

Reply:
ho, it's just to separate things in display. So in between other string not necessary to add new line. But it's just cosmetic 😅

Reply:
trying to clean thing to preprare an upgrade option inside the install script. Your `NoClean` option is a good starting point 😊